server.go

```diff
@@ -2687,19 +2687,18 @@ func (s *server) ConnectToPeer(addr *lnwire.NetAddress, perm bool) error {
 
 	// Peer was not found, continue to pursue connection with peer.
 
-	// If there's already a pending connection request for this pubkey,
```

Comment:
good catch! this comment was outdated for sure. i think we used to return an error here, and then replaced it with a log but looks like the comment never got updated.